internal/tofumigrate/tofumigrate.go

```diff
@@ -53,6 +54,9 @@ func MigrateStateProviderAddresses(config *configs.Config, state *states.State)
 			if resource.ProviderConfig.Provider.Hostname == "registry.terraform.io" && !referencedInConfig {
 				resource.ProviderConfig.Provider.Hostname = tfaddr.DefaultProviderRegistryHost
 			}
+			if resource.ProviderConfig.Provider.Namespace == "hashicorp" && !referencedInConfig {
```

Comment:
From discover during our ad-hoc testing from yesterday, is this still needed?

Reply:
This is not needed, let's roll this file back.